sc-72333-remove-rss-symbol-link-in-the-pennylane-ai-docs

ashishks0522 opened the pull request:
## Changes
- [sc-72333-remove-rss-symbol-link-in-the-pennylane-ai](https://app.shortcut.com/xanaduai/story/72333/remove-rss-symbol-link-in-the-pennylane-ai-footer)


## Related PR
- https://github.com/XanaduAI/pennylane.ai-react/pull/1759

Mandrenkov commented:
Hey @ashishks0522, a couple of things:
1. It looks like the [Read the Docs build](https://readthedocs.com/projects/xanaduai-pennylane-sphinx-theme/builds/2500455/) is failing because of a NumPy version compatibility. You should be able to fix this by adding `numpy==1.26.4` to the `doc/requirements.txt` file.
2. We should update the release notes and bump the version of the PennyLane Sphinx Theme package in `pennylane_sphinx_theme/_version.py` to `0.5.8`.

Mandrenkov reviewed (changes requested):
Thanks @ashishks0522, looks like the [RSS icon is gone now](https://xanaduai-pennylane--65.com.readthedocs.build/projects/sphinx-theme/en/65/)! 🎉 

![image](https://github.com/user-attachments/assets/d83c4fca-b5b7-4021-8ac6-a05d2aef0962)


There are, however, a couple of things we should do before merging:
1. Update the changelog and Python package version to `v0.5.8`.
2. Update the copyright date in the footer to `2024`.

![image](https://github.com/user-attachments/assets/8c36026c-b495-4398-b9a6-e815fee9534f)

ashishks0522 commented:
@Mandrenkov I have updated the version and added a tech debt [ticket](https://app.shortcut.com/xanaduai/story/74407/make-footer-year-dynamic-in-pennylane-docs-website) to look into making copyright year dynamic.

Mandrenkov commented on .github/CHANGELOG.md:
```diff
@@ -1,10 +1,17 @@
+### Improvements
+
+* Removed the rss feed icon from the footer
+  [(#63)](https://github.com/PennyLaneAI/pennylane-sphinx-theme/pull/65)
```
# Typo
```suggestion
  [(#65)](https://github.com/PennyLaneAI/pennylane-sphinx-theme/pull/65)
```

Mandrenkov commented on .github/CHANGELOG.md:
```diff
@@ -1,10 +1,17 @@
+
+### Improvements
+
+* Removed the rss feed icon from the footer
```
# Format
```suggestion
* Removed the RSS feed icon from the footer.
```

Mandrenkov reviewed (approved):
Thanks @ashishks0522, LGTM! 🚀
